packages/sysweb3-keyring/test/keyring-manager.spec.ts

```diff
@@ -62,13 +62,24 @@ describe('', () => {
 
   // * addNewAccount
   it('should add a new account', async () => {
-    const account = await keyringManager.addNewAccount(undefined);
-    expect(account.label).toBe('Account 2');
+    const account2 = await keyringManager.addNewAccount(undefined);
+    expect(account2.label).toBe('Account 2');
 
     const wallet = keyringManager.getState();
     expect(wallet.activeAccount.id).toBe(1);
   });
 
+  it('should derivate a new account with specific address', async () => {
+    const account2 = await keyringManager.addNewAccount();
+    expect(account2.address).toBe('0xd5e66a5d61690dd4d6675d1e9eb480ddd640fe06');
```

Comment:
Could you create the new keyRing here with the expected seed already so this test doesn't fail if someone uses a different seed on the env file?
Suggestion seed:
gauge gauge gauge gauge gauge gauge gauge gauge gauge gauge gauge gauge